AdminCore.ScheduledJobs/ScheduleRunner.cs

```diff
@@ -0,0 +1,73 @@
+﻿using AdminCore.Common.Interfaces;
+using AdminCore.DAL;
+using AdminCore.DTOs;
+using Quartz;
+using Quartz.Impl;
+using System;
+
+namespace AdminCore.ScheduledJobs
+{
+  public class ScheduleRunner : IScheduleRunner, IDisposable
+  {
+    private readonly IScheduler _scheduler;
+
+    public ScheduleRunner(IDatabaseContext databaseContext)
+    {
+      var factory = new StdSchedulerFactory();
+      _scheduler = factory.GetScheduler().Result;
+      _scheduler.Start();
+
+      _scheduler.JobFactory = new DatabaseJobFactory(databaseContext);
```

Comment:
So this is a singleton and the same DB context will be used for each job. This was fine when the last approach used LikeDB. Here we might get unit of work issues